lib/getPaths.js

```diff
@@ -6,6 +6,7 @@
 "use strict";
 
 module.exports = function getPaths(path) {
+	if (path === "/") return { paths: ["/"], seqments: [""] };
```

Comment:
have find a small bug where several same results were returned.
without this line `{ paths: ["/", "/"] }` returned